src/dune_console/backend_intf.ml

```diff
@@ -0,0 +1,19 @@
+open Stdune
+
+module type S = sig
```

Comment:
I think it would be a good idea to document these.

Reply:
It's already documented in the public toplevel module `Dune_console`.